packages/paste-core/components/chat-composer/src/ChatComposerContainer.tsx

```diff
@@ -0,0 +1,76 @@
+import type { BoxProps } from "@twilio-paste/box";
+import { Box, safelySpreadBoxProps } from "@twilio-paste/box";
+import { ThemeShape } from "@twilio-paste/theme";
+import * as React from "react";
+
+import { ChatComposerContext } from "./ChatComposerContext";
+
+const Styles = {
+  default: {},
+  contained: {
+    borderWidth: "borderWidth10" as ThemeShape["borderWidths"],
+    borderStyle: "solid",
+    borderRadius: "borderRadius30" as ThemeShape["radii"],
+    borderColor: "colorBorderWeaker" as ThemeShape["borderColors"],
+    boxShadow: "shadowBorderLow" as ThemeShape["shadows"],
```

Comment:
suggestion/question: I think it still needs backgroundColor: colorBackgroundBody